homeassistant/components/apcupsd/const.py

```diff
@@ -4,3 +4,6 @@
 
 DOMAIN: Final = "apcupsd"
 CONNECTION_TIMEOUT: int = 10
+
+# Field name of last self test retrieved from apcupsd.
+LASTSTEST: Final = "laststest"
```

Comment:
Please separate words with underscore in variable names, `LAST_S_TEST`.

Reply:
This is more to be consistent with the field name from the APC UPS Daemon, which names it exactly as `LASTSTEST`. Having consistency here feel more natural to me (especially considering this is parallel to the other fields). Let me know if you think otherwise! 

(Or, perhaps `LAST_SELF_TEST_FIELD` to break the consistency but gain better clarity)

Reply:
As long as we use snake_case to separate words it's ok. It's too hard to read `LASTSTEST`.

Reply:
Sounds good! A PR is created: https://github.com/home-assistant/core/pull/124668